activejob/test/support/integration/adapters/resque.rb

```diff
@@ -3,11 +3,10 @@
 module ResqueJobsManager
   def setup
     ActiveJob::Base.queue_adapter = :resque
-    Resque.redis = Redis::Namespace.new "active_jobs_int_test", redis: Redis.new(url: "redis://:password@127.0.0.1:6379/12", thread_safe: true)
+    Resque.redis = Redis::Namespace.new "active_jobs_int_test", redis: Redis.new(url: "redis://127.0.0.1:6379/12", thread_safe: true)
     Resque.logger = Rails.logger
     unless can_run?
-      puts "Cannot run integration tests for resque. To be able to run integration tests for resque you need to install and start redis.\n"
-      exit
+      abort "Cannot run integration tests for resque. To be able to run integration tests for resque you need to install and start redis.\n"
```

Comment:
I think this is intended to be a 'soft' failure so that contributors can run the suite without needing all the queue adapters' dependencies set up.

Maybe we should make it a hard fail `if ENV["CI"]`?

Reply:
:+1: Fixed to hard fail only CI.